### PR TITLE
Added contribution graph to user model

### PR DESCRIPTION
**Pull request by chrisngyn**
Title

### CODE REVIEW

File: data_collection/contribution_chart.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
+from general.user import username, user_token
+from general.static import endpoints
+import json
+import requests
+
+def get_contribution_chart(user):
+    chart_query = f"""query {{
+        user(login: "{user}") {{
+            contributionsCollection(from: "2020-06-01T00:00:00Z", to: "2020-08-24T19:30:46Z") {{
```
Comment (kbanc) [inline data_collection/contribution_chart.py:9]: can you use the times defined in general_static

File: data_collection/contribution_chart.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        }}
+    }}"""
+
+    response = requests.post(
+        endpoints["github"],
+        headers={"Authorization": f"token {user_token}"},
+        json={"query": chart_query}
+    )
+
+    return json.dumps(response.json())
```
Comment (kbanc) [inline data_collection/contribution_chart.py:30]: also did you want to be more selective about what part of the response you dump? I think maybe returning the `response["data"]["user"]["contributionsCollection"]["contributionCalendar"]` may make it easier to manipulate later

File: data_collection/contribution_chart.py (hunk 1 above)
Comment (chrisngyn) [reply data_collection/contribution_chart.py:9]: RIGHTTT my bad I completely forgot about the static variables 😅

File: data_collection/contribution_chart.py [hunk 3]
```diff
@@ -0,0 +1,35 @@
+from general.user import username, user_token
+from general.static import endpoints
```
Comment (kx-chen) [inline data_collection/contribution_chart.py:2]: Should these be prefixed with `data_collection.`?

**Pull request merged**